### PR TITLE
Added Feedback for SAA

### PR DESCRIPTION
**Pull request by rhobinjay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,27 @@ The consolidated contents are comprised of both free and paid services that were
...
+   * **Type:** Udemy course
+   * **From:** Stephane Maarek
+   * **Notes:**
+     * First few courses started with general knowledge about AWS before jumping into architectural concepts. Really helpful to try looking for unfamiliar technical words online before continuing the lecture
+     * Important things are well highlighted and it even mentioned on other lectures which I find it easy to remember
+     * Includes a summary of a topic in the end to see the concept in a bigger picture and it highlights what needs to remember first before understanding the details.
+     * Ideas and designs are well illustrated as the diagrams are easily understandable
+     * Questions posted on the community are not answered right away but it took within the day for some at least. Although in Udemy, you can search for the question first before posting one.
+     * Pronunciation is sometimes unclear and there are sudden cuts to only few of the lectures. Subtitles are recommended.
+* [AWS Certified Solutions Architect Associate Practice Exams](https://www.udemy.com/course/aws-certified-solutions-architect-associate-practice-tests-k/) :dollar: :star:
```
Comment (maronavenue) [inline README.md:84]: this is really great. i've only tried practice exams from `Jon Bonso`, so this provides a totally different perspective.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,27 @@ The consolidated contents are comprised of both free and paid services that were
...
+   * **Type:** Udemy course
+   * **From:** Neal Davis
+   * **Notes:**
+     * Contains 6 sets of test with 65 questions each, multiple choice and multiple answer. Then each test has 120mins time to finish. This is to simulate the real certification test.
+     * Quality of questions are great and almost give you the same feeling when doing actual cert exam
+     * Results will be provided after the test with a high level summary to different knowledge areas
+     * All choices are thoroughly explained why it is correct and why not.
+     * Expect mental stress and work-out when decided to do the practice test.
+     * Prepare around 2 weeks to complete all tests with the understanding of correct and incorrect answers from the question.
```
Comment (maronavenue) [inline README.md:93]: +1: practice exams should be given ample time to process and complete.

File: README.md (hunk 1 above)
Comment (maronavenue) [inline README.md:84]: thought the "-k" at the end of the url was a typo but it's just how it is on udemy lol.

**Review by maronavenue — APPROVED** [review]
the general format looks good overall. thanks for adding key feedbacks based on your real experiences on these courses. i like the neutral and objective notes to set proper expectations for the learners.

**Comment by rhobinjay** [comment]
Thanks @maronavenue!

**Pull request merged**